content/examples/case-studies.md

```diff
@@ -21,3 +21,5 @@ These links can serve as inspiration for your own research or to further your un
 * [**Open Source Design Processes at Ushahidi**](https://docs.ushahidi.com/platform-developer-documentation/design/design-process):
   This documentations work at Ushahidi was funded by DIAL to improve the readability and inclusivity of Ushahidi's documentation to encourage contribution across functions and skills.
   It documented good amounts of examples of how a design function within a moderately resourced NGO OSS org could be done.
+* [**Sustain Open Source Design Podcast**](/design/examples/podcasts/):
```

Comment:
We can use a [`relref`](https://gohugo.io/functions/relref/) function to create an internal link to the page. This would also continue to work if a page redirect was set up or the page was translated to another language:

```suggestion
* [**Sustain Open Source Design Podcast**]({{< relref . "podcasts" >}}):
```